fmf/base.py

```diff
@@ -438,7 +438,7 @@ def update(self, data):
 
     def adjust(self, context, key='adjust', undecided='skip',
                case_sensitive=True, decision_callback=None,
-               additional_rules=None):
+               additional_rules=None, additional_rules_callback=None):
```

Comment:
Please, add also a protocol type similar to `AdjustCallback`.

Reply:
Added in 545d2e2